packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -811,6 +811,9 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 					for (const [id, entry] of this.pendingBlobs) {
 						if (!localBlobs.has(entry)) {
 							localBlobs.add(entry);
+							if (!entry.opsent) {
+								this.sendBlobAttachOp(id, entry.storageId);
```

Comment:
Maybe add a comment here about why we need to send the blob attach op here, and not elsewhere